src/EnergyPlus/WeatherManager.cc

```diff
@@ -3731,6 +3750,27 @@ namespace WeatherManager {
         return (fmod(interpAng, 360.)); // fmod is float modulus function
     }
 
+    Real64 CalcSkyEmissivity(int ESkyCalcType, Real64 OSky, Real64 DryBulb, Real64 DewPoint, Real64 RelHum){
+        //Calculate Sky Emissivity
+        Real64 ESky;
+        double TDewC = min(DryBulb, DewPoint);
+        double SatPress = PsyPsatFnTemp(DryBulb) * 0.01;
+        double PartialPress = RelHum * SatPress;
+        double TDewK = TDewC + TKelvin;
```

Comment:
Marking these as `const` would be best-practice

Reply:
Why assign all of these here when only some are used depending on the sky model type? Seems these should all be moved within the respective sky model where they are used.

Reply:
That's also true @mjwitte.  Though PartialPress (and therefore SatPress) is used by two of them, and TDewC is implicitly used in two of them, so there would be some duplication.  I'm OK with leaving it like this or putting the initialization into each IF block.